docs/09-dev-guide.md

```diff
@@ -39,22 +40,32 @@ into the scripts directory so that the scripts work.
 ./scripts/build/buildForTest.sh
 ```
 
-maven (removing the '-Pquick' option will cause the unit and integration tests
+Maven (removing the '-Pquick' option will cause the unit and integration tests
 to run):
 
 ```bash
 cd java
 mvn clean install -Pquick
 ```
 
+## Exploring the code base
+
+Look in the [design document](10-design.md) for an idea of what to expect in the project. The elements of the design
+largely correspond to Maven modules. Core or common modules are used to separate shared model code from integrations
+with libraries which are not needed by all components of the system.
+
+If you'd like to look at how the modules relate to one another in terms of their dependencies, in the `build` module
+there's a main method in the class `sleeper.build.maven.ShowInternalDependencies` to display the internal dependencies
```

Comment:
Do we want to make the build module so that this class can easily be run from the command line?

Reply:
Yup, we can run it through Maven like with `scripts/dev/generatePropertiesTemplates.sh`.

Reply:
I've added that now.